### ISSUE TITLE
Is this code currently limited to a single string?

### ISSUE THREAD

**Issue by zestysoft** [rp-/tigo-exporter#4]
I apologize in advance -- I'm not familiar with Rust.

re this code:
https://github.com/rp-/tigo-exporter/blob/2df8d2ef55d9715c8c55fd62a76748ee00dc70d8/src/main.rs#L74

Does that mean it only recognizes a single string of panels?  I've got an additional B string with more panels.

**Comment by rp-** [comment]
yes it will, I only have one string and had no other test-data or information how other strings look like
